nvm.sh

```diff
@@ -2446,13 +2448,26 @@ nvm() {
         nvm_err "wget: not found"
       fi
 
-      for tool in git grep awk sed cut basename rm mkdir xargs; do
+      local TEST_TOOLS
+      TEST_TOOLS='git grep awk'
+      if [ 'darwin' != "${NVM_OS}" ] && [ 'freebsd' != "${NVM_OS}" ]; then
+        TEST_TOOLS="${TEST_TOOLS} sed cut basename rm mkdir xargs"
```

Comment:
I feel like maybe instead of omitting them, we should still be able to print out *something* - it would be good, on these OS's, to know that the commands exist, and the path to them.

Reply:
I think maybe this "improvement" can be done later? 

This PR is trying to "fix" the exception of `nvm debug` on macOS, and has been opened for more than 1 year, what do you think?

Reply:
The current state is that it either prints out an error about not understanding a version arg, or it prints out that the binary is not found - this PR removes the information of "installed or not" - so sure, we could merge this and do a followup, but is there a reason that's more advantageous than to update this PR now?

Reply:
Just think it's broken for a long time, it'll be great to have it work normally on macOS.

Reply:
I'll give it a try to see if it's easy or complex to have an improvement here immediately, as the time to work on this is not guarantee, that's why I think it could be merged first, and come up with an improvement in future.

Reply:
Revised again, should be good now ... @ljharb please take a look, thanks